src/components/Sidebar/Sidebar.tsx

```diff
@@ -112,6 +112,23 @@ export const Sidebar = ({ mentors, handleModal }: SidebarProps) => {
           handleModal={openModal}
         />
       </SidebarNav>
+      <a
+        href="https://vercel.com/?utm_source=[coding-coach]&utm_campaign=oss"
+        aria-label="Vercel"
+        target="_blank"
+      >
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="212"
+          height="44"
+          viewBox="0 0 212 44"
+        >
+          <path
```

Comment:
What do you think about saving this in an `.svg` file and `import` it like we do here

https://github.com/Coding-Coach/find-a-mentor/blob/7dfd3223dcb9dd1dcd532ce84da17ee659a68098/src/PageNotFound.tsx#L1

Reply:
Updated with the import ...thank you for calling that out